src/components/Navbar/index.tsx

```diff
@@ -124,6 +128,9 @@ const Navbar: React.FC<Props> = ({ onToggleMode }) => {
 						<a href={socials.linkedin}>
 							<LinkedIn />
 						</a>
+						<a href={socials.linkedin}>
+							<Twitter />
```

Comment:
**Suggestion:** The Twitter link is incorrectly set to LinkedIn's URL. It should be set to Twitter's URL.
```suggestion
      <a href={socials.twitter}>
        <Twitter />
      </a>
```